src/package.json

```diff
@@ -1,22 +1,34 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "10.1.5",
+  "version": "10.1.6",
   "description": "An adapter for connecting to Mojaloop API enabled switches.",
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
-    "test": "jest --ci --reporters=default --reporters=jest-junit --env=node test/unit",
-    "test:int": "jest --ci --reporters=default --reporters=jest-junit --env=node test/integration --forceExit"
+    "test": "jest --ci test/unit",
+    "test:int": "jest --ci --collectCoverage=false test/integration --forceExit"
   },
   "author": "Matt Kingston, James Bush, ModusBox Inc.",
+  "contributors": [
+    "Paweł Marzec <pawel.marzec@modusbox.com>"
+  ],
   "license": "Apache-2.0",
   "licenses": [
     {
       "type": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0"
     }
   ],
+  "husky": {
```

Comment:
Strongly protest the use of pre-commit hooks like this, as I think with a CICD pipeline in place, they incur more cost than they provide in benefit.

Reply:
I would insist to automate the control of code quality as early in the development workflow as possible.

Reply:
I like the idea but I would prefer this to be a `pre-push` hook instead to speed up development cycle, as there can be scenarios that you want to commit locally many times before going through quality control of your code.

Reply:
> I would insist to automate the control of code quality as early in the development workflow as possible.

I think it's a bit too prescriptive of a workflow. E.g., if you could dictate we all used VS Code and then dictate with a file that we all did a test run and a lint on every save, you'd probably be in favor of that. I wouldn't though because there's a certain point that you have to step back and let individual developers determine how they'd best like to get to the same conclusion, which is quality code that passes peer-review.

Reply:
I somewhat agree with this - especially in the use of the `npm-check-updates` that we have in other repos (which isn't in scope here).

My only other thing to add is this: `git commit -n` skips the pre-commit hooks, so you can somewhat get the best of both worlds.

Reply:
@vbarzokas I moved unit testing to pre-push hook and leave linting - but only staged files - in pre-commit hook.

Reply:
Again, the change to a pre-push hook was made to be prescriptive of one person's particular workflow at the expense of another's--it does not serve to solve anything universally.

Reply:
I am also of the opinion that the branch protections and CI should take the responsibility of "enforcing" the quality of master branch. I think that although some may find git hooks useful, others will find them intrusive. This is a community though but the standards do not appear to specify one way or the other. I suggest we look for the best compromise.